### PR TITLE
sleepwatcher: fix service declaration by using service block

### PR DESCRIPTION
**Pull request by nicerloop**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/sleepwatcher.rb [hunk 1]
```diff
@@ -36,32 +36,21 @@ def install
...
-
-    # Write the launchd scripts
-    inreplace Dir["config/*.plist"], "/usr/local/sbin", HOMEBREW_PREFIX/"sbin"
-
-    inreplace "config/de.bernhard-baehr.sleepwatcher-20compatibility.plist",
-      "/etc", etc/"sleepwatcher"
-
-    prefix.install Dir["config/*.plist"]
+  service do
+    run [HOMEBREW_PREFIX/"sbin/sleepwatcher", "-V", "-s", "#{Dir.home}/.sleep", "-w", "#{Dir.home}/.wakeup"]
```
Comment (SMillerDev) [inline Formula/sleepwatcher.rb:42]: This should work 

```suggestion
    run [sbin/"sleepwatcher", "-V", "-s", "#{Dir.home}/.sleep", "-w", "#{Dir.home}/.wakeup"]
```

Comment (nicerloop) [reply Formula/sleepwatcher.rb:42]: Does not work:
Error: Failed to install service files
Error: undefined local variable or method `sbin' for #<Homebrew::Service:0x00007fb6dbaea800>
Did you mean?  bin
Please report this issue:
  https://docs.brew.sh/Troubleshooting
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/sleepwatcher.rb:42:in `block in <class:Sleepwatcher>'
/usr/local/Homebrew/Library/Homebrew/service.rb:169:in `instance_eval'
/usr/local/Homebrew/Library/Homebrew/service.rb:169:in `command'
/usr/local/Homebrew/Library/Homebrew/extend/os/mac/caveats.rb:40:in `plist_caveats'
/usr/local/Homebrew/Library/Homebrew/caveats.rb:43:in `caveats'
/usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.3_2/lib/ruby/2.6.0/forwardable.rb:224:in `empty?'
/usr/local/Homebrew/Library/Homebrew/formula_installer.rb:744:in `caveats'
/usr/local/Homebrew/Library/Homebrew/formula_installer.rb:795:in `finish'
/usr/local/Homebrew/Library/Homebrew/install.rb:304:in `install_formula'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:207:in `block in install'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:205:in `each'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:205:in `install'
/usr/local/Homebrew/Library/Homebrew/brew.rb:122:in `<main>'

**Comment by nicerloop** [comment]
Hi, what do I need to do to resolve the 'missing license'?

**Comment by cho-m** [comment]
> Hi, what do I need to do to resolve the 'missing license'?

That is related to the `license` stanza https://docs.brew.sh/License-Guidelines

If you grab a copy of source code, you can usually check on License information used. In this case, it is probably `GPL-3.0-or-later`.

From source,
- ReadMe.rtf:

    > Copyright (C) 2002-2019 Bernhard Baehr (bernhard.baehr@gmx.de)\
    > \
    > This program is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation; either version 3 of the License, or (at your option) any later version.\

- sources/sleepwatcher.c
    ```
     *	sleepwatcher.c - sleep mode watchdog program
     *
     *	This program is free software: you can redistribute it and/or modify
     *	it under the terms of the GNU General Public License as published by
     *	the Free Software Foundation, either version 3 of the License, or
     *	(at your option) any later version.
    ```

The homepage (https://www.bernhard-baehr.de/) also links to copy of GPLv3 license, though you would still need to check source code to distinguish between `GPL-3.0-only` and `GPL-3.0-or-later`

File: Formula/sleepwatcher.rb [hunk 2]
```diff
@@ -4,6 +4,8 @@ class Sleepwatcher < Formula
   url "https://www.bernhard-baehr.de/sleepwatcher_2.2.1.tgz"
   sha256 "4bf1656702167871141fbc119a844d1363d89994e1a67027f0e773023ae9643e"
 
+  license "GPL-3.0-or-later"
```
Comment (SMillerDev) [inline Formula/sleepwatcher.rb:7]:
```suggestion
  sha256 "4bf1656702167871141fbc119a844d1363d89994e1a67027f0e773023ae9643e"
  license "GPL-3.0-or-later"
```

File: Formula/sleepwatcher.rb [hunk 3]
```diff
@@ -36,32 +38,21 @@ def install
...
-
-    # Write the launchd scripts
-    inreplace Dir["config/*.plist"], "/usr/local/sbin", HOMEBREW_PREFIX/"sbin"
-
-    inreplace "config/de.bernhard-baehr.sleepwatcher-20compatibility.plist",
-      "/etc", etc/"sleepwatcher"
-
-    prefix.install Dir["config/*.plist"]
+  service do
+    run [HOMEBREW_PREFIX/"sbin/sleepwatcher", "-V", "-s", "#{Dir.home}/.sleep", "-w", "#{Dir.home}/.wakeup"]
```
Comment (SMillerDev) [inline Formula/sleepwatcher.rb:44]:
```suggestion
    run [opt_sbin/"sleepwatcher", "-V", "-s", "#{ENV["HOME"]}/.sleep", "-w", "#{ENV["HOME"]}/.wakeup"]
```

**Comment by nicerloop** [comment]
Hi, can someone review this PR? Thanks a lot.

**Review by SMillerDev — DISMISSED** [review]

**Comment by SMillerDev** [comment]
@nicerloop can you rebase this?

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**